lmdeploy/cli/serve.py

```diff
@@ -206,7 +200,6 @@ def gradio(args):
         from lmdeploy.archs import autoget_backend
         from lmdeploy.messages import (PytorchEngineConfig,
                                        TurbomindEngineConfig)
-        from lmdeploy.model import ChatTemplateConfig
```

Comment:
`PytorchEngineConfig` and `TurbomindEngineConfig` do not have `model_name`, may need to update line 219 and line 228